derive_client/clients/base_client.py

```diff
@@ -833,3 +838,258 @@ def transfer_from_subaccount_to_funding(self, amount: int, asset_name: str, suba
             condition=_is_final_tx,
             transaction_id=withdraw_result.transaction_id,
         )
+
+    def transfer_position(
+        self,
+        instrument_name: str,
+        amount: float,
+        limit_price: float,
+        from_subaccount_id: int,
+        to_subaccount_id: int,
+        position_amount: float = None,
```

Comment:
Please be explicit amount to be transfered,
Never allow this to be none.
In the Cli, perform this retrieval if it is none, allowing the cli to basically perform the retrieval